oeps/oep-0019-bp-developer-documentation.rst

```diff
@@ -52,6 +52,20 @@ This proposal focuses on long-term developer *Documentation* needs. It does not
 Decisions
 *********
 
+* We will use `reStructuredText (rST)`_ for all developer documentation (including in docstrings).  See the :ref:`Format Rationale` section below for more info.
+
+* The `Developer Docs`_ section of docs.edx.org will serve as the index for all repo specific and system wide developer docs.
```

Comment:
As detailed in https://github.com/edx/open-edx-proposals/pull/232#discussion_r689832342, I think we are missing a more important bullet about `edx-developer-docs`, which would provide more clarity about where the index really lives.

Reply:
@robrap I'm a little confused about this so maybe you can provide some more clarity.  The developer docs section _is_ the pages in the edx-developer-docs repo.  I don't get the point about a separate bullet and what it would say.  Maybe the useful thing here is to add a second sentence that explains that the developer docs section is backed by a single repo and link to that repo?

Reply:
I see. I think that bullet is confusing, and something like the following would be a more clear replacement:
```
* The repo `edx-developer-docs` will serve as the main index for all repo specific and system wide developer docs.
```

**Optionally,** if we think it is important to mention docs.edx.org at all in the OEP (which I don't really think so), we could have an additional bullet like:
```
* The "Developer Docs" section of docs.edx.org will link to the `edx-developer-docs` published docs.
```

Reply:
So Nimisha and I were talking about this during the hackathon.  We came up with the idea of renaming `open-edx-proposals` to `open-edx-developer-docs` and co-locating our developer docs content with the OEPs in this repo.  Then getting rid of `edx-developer-docs`, since it has barely any content in it to begin with.  It felt like a simplification/optimization to start getting all our developer docs in one place.

I was going to try to do that as a hackathon project, but was squeamish about:

1. Needing to get a fully functioning Sphinx setup in this repo.
2. Renaming it without first thinking through what links I might break.

I went through an exercise of transferring all of the `edx-documentation` developers content over to `edx-developer-docs` and had to fix a few things along the way.  But I think I have a version of that content that's pretty portable at this point and could be moved around if we decided to go with moving it here. 

This is maybe a longer chat and I'm probably forgetting some nuance of our conversation.